src/lex/cpp.rs

```diff
@@ -1215,19 +1257,33 @@ mod tests {
             _ => panic!("not a keyword: {:?}", token),
         }
     }
+    fn _is_not_whitespace(res: &CompileResult<Token>) -> bool {
+        !matches!(res, Ok(Token::Whitespace(_)))
+    }
```

Comment:
Why is there both this function and `is_not_whitespace`?

Reply:
I was being dumb and applied the second one in a different order so I though I needed two with different signatures Changed in coming commit.